From aba69d879633a9a958d6f797700e4d8a860c317d Mon Sep 17 00:00:00 2001
From: Davide Gerhard <rainbow@irh.it>
Date: Sat, 14 Nov 2020 15:55:41 +0100
Subject: [PATCH] Fix PortAudio with GNU Radio 3.8 on macOS/Darwin platform

this is needed if the file system is case-sensitive
---
 CMakeLists.txt | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5c1fca03..e3ea507d 100644
--- CMakeLists.txt
+++ CMakeLists.txt
@@ -184,7 +184,11 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
     endif()
 
     if(${OSX_AUDIO_BACKEND} MATCHES "Portaudio")
-        find_package(Portaudio REQUIRED)
+        if(Gnuradio_VERSION VERSION_LESS "3.8")
+            find_package(Portaudio REQUIRED)
+        else()
+            find_package(PORTAUDIO REQUIRED)
+        endif()
         add_definitions(-DWITH_PORTAUDIO)
         unset(PULSEAUDIO_FOUND CACHE)
         unset(PULSEAUDIO_INCLUDE_DIR CACHE)
